### ISSUE TITLE
Roomscale bow arrows somethimes collide with the player

### ISSUE THREAD

**Issue by Dr-WeiAL** [ferriarnus/VivecraftForge#127]
When I use the bow, the trajectory often disappears, but it actually fires out and can hit the target aimed at
This makes it impossible to observe the parabola and the point of fall

Version: 1.18.2 Forge
System: Windows 10
VR device: Rift S

**Comment by fayer3** [comment]
what do you mean with `the trajectory often disappears`?
is the arrow not there when it flies?

**Comment by fayer3** [comment]
there is no visual aiming aid, if you mean that

**Comment by Dr-WeiAL** [comment]
No, when using the bow, the arrow has a high probability of bouncing back in front of the eyes, and then disappeared, but the arrow actually shot out, but can not see the trajectory of the arrow, only see the arrow in the landing position

**Comment by Dr-WeiAL** [comment]
I uploaded the video in Discord
Incidentally, I didn't have this problem when I added vivecraft to the modpack, but it occurs when no other mods are added

**Comment by fayer3** [comment]
mentioned video, just for future reference
https://cdn.discordapp.com/attachments/923042654537805876/1083743721520123944/2023-03-10_21-22-41_3.mp4

**Comment by fayer3** [comment]
discord message link, since the video link expired https://discord.com/channels/923042654537805874/923042654537805876/1083743722182803496

**Comment by fayer3** [comment]
maybe fixed with 22f6cd5bb41316cd2ff2444ae7a136028c655b3d
if this still happens with 1.3.0 which will come out soon, feel free to reopen this
